### ISSUE TITLE
Review NVDA browse mode up/down arrow command assertions in Rating Radio Group V25.04.22

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at#1365]
As prescribed in #1246, assess whether assertions need to be changed for NVDA's browse mode up/down arrow commands or whether the commands should be removed from some tests.

## Metadata

- Test Review Page: [Link](https://aria-at-.w3.org/test-review/163646)

**Comment by mcking65** [comment]
Reviewed the following commands and assertions in [Rating Radio Group Test Plan V25.04.22](https://aria-at.w3.org/test-review/163646):
* up/down arrows in browse mode in tests 1 through 4
* Insert+up in browse mode in test 14 and 15
The commands and assertions are consistent with:
* Comparable JAWS commands and assertions for the same tests
* Comparable commands and assertions in:
    * [Radio Group Example Using aria-activedescendant](https://aria-at.w3.org/report/163665/targets/520)
    * [Radio Group Example Using Roving tabindex](https://aria-at.w3.org/report/163667/targets/528)

At this time, the best path forward seems to be to keep the commands and assertions as they are and report excess verbosity consistent with how it was reported for [Radio Group Example Using aria-activedescendant](https://aria-at.w3.org/report/163665/targets/520). This will trigger an issue to be raised that can drive an appropriate discussion of default behavior with NVAccess.

It is worth noting that NVDA does not generate the same verbosity for [Radio Group Example Using Roving tabindex](https://aria-at.w3.org/report/163667/targets/528).
